tests/lib/schema_validation_test.py

```diff
@@ -170,6 +181,30 @@ def test_valid_without_auto_neg_and_missing_speed(self, default_data):
 
         libnmstate.validator.validate(default_data)
 
+    @parametrize_sriov_valid_total_vfs
+    def test_valid_with_sriov_total_vfs(self, default_data, valid_values):
+        default_data[Interface.KEY][0].update(
+            {
+                Interface.TYPE: InterfaceType.ETHERNET,
+                SRIOV.CONFIG_SUBTREE: {SRIOV.TOTAL_VFS: valid_values},
+            }
+        )
+        libnmstate.validator.validate(default_data)
+
+    @parametrize_sriov_invalid_total_vfs
+    def test_over_maximum_total_vfs_is_invalid(
+        self, default_data, invalid_values
+    ):
+        default_data[Interface.KEY][0].update(
+            {
+                Interface.TYPE: InterfaceType.ETHERNET,
+                SRIOV.CONFIG_SUBTREE: {SRIOV.TOTAL_VFS: invalid_values},
+            }
+        )
+
+        with pytest.raises(js.ValidationError):
+            libnmstate.validator.validate(default_data)
```

Comment:
shouldn't you verify the error msg as well ? What I had suggested in a previous comment was for you to use the *match* keyword instead, no for your to stop checking that.

I meant something like:
```
with pytest.raises(ValueError, match=r".* 123 .*"):
    myfunc()
```

From [0] I get the understanding this is pytest's way of asserting on py3.

[0] - https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions

Reply:
That seems fine. Done! Thanks :-)